src/clj/rems/css/styles.clj

```diff
@@ -1,8 +1,11 @@
 (ns rems.css.styles
   "CSS stylesheets are generated by garden automatically when
   accessing the application on a browser. The garden styles can also
-  be manually compiled by calling the function
-  rems.css.styles/generate-css"
+  be manually compiled by calling the function `rems.css.styles/screen-css`.
+
+  For development purposes with Figwheel the styles are rendered to
+  `env/dev/resources/public/css/screen.css` whenever this ns is evaluated
+  so that Figwheel can autoreload them."
```

Comment:
is there no other way to hook into figwheel autoreload than the filesystem?

Reply:
No